mobile/lib/components/ra_player/ra_player_handler.dart

```diff
@@ -108,18 +181,16 @@ class AudioPlayerHandler extends BaseAudioHandler with SeekHandler {
     return PlaybackState(
       controls: [
         // MediaControl.rewind,
-        // if (_player.playing) MediaControl.pause else MediaControl.play,
+        if (mediaKind.value == MediaKind.recording)
+          if (_player.playing) MediaControl.pause else MediaControl.play,
         // MediaControl.stop,
         // MediaControl.fastForward,
         if (_player.playing) MediaControl.stop else MediaControl.play,
       ],
-      // systemActions: const {
-      //   MediaAction.seek,
-      //   MediaAction.seekForward,
-      //   MediaAction.seekBackward,
-      // },
+      systemActions: {
+        if (mediaKind.value == MediaKind.recording) MediaAction.seek,
+      },
       androidCompactActionIndices: const [0],
```

Comment:
accordingly to changes above (although on my device it made no difference at all)

```suggestion
      androidCompactActionIndices:
          mediaKind.value == MediaKind.recording ? [1] : [0],
```